code-studio/src/js/initApp/initApp.js

```diff
@@ -138,6 +143,11 @@ window.apps = {
         }
       }
     })(appOptions.level);
+
+    // Previously, this was set by dashboard based on route and user agent. We
+    // stopped being able to use the user agent on the server, and thus try
+    // to have the same logic on the client.
+    appOptions.noPadding = !appOptions.isLegacyShare && isMobile();
```

Comment:
It looks like noPadding would also have been set on embed levels in the past. As a result, I originally had `|| appOptions.embed` here. However, when looking at an embed level it looked identical with or without noPadding set, so I decided to simplify this. This should have no chance of regressing embed levels, since they would currently always have noPadding === false.